exercises/twelve-days/src/test/java/TwelveDaysTest.java

```diff
@@ -120,18 +120,21 @@ public void testVerseTwelve() {
     @Ignore("Remove to run test")
     @Test
     public void testMultipleVerses() {
+/*
```

Comment:
Awesome; in light of that consensus, please update this PR to include stubs of both methods and no commented-out tests; I'll update the policies doc right now.